downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
+
+If you are using file-based storage and your installation scenario includes {HubName}, ensure that you change the `ReadWriteOnce` default storage option for {OperatorPlatform} to `ReadWriteMany`.
+
+{HubNameStart} requires `ReadWriteMany` storage, Azure Blob storage, or an s3-compliant storage for operation because multiple pods responsible for upload and download of content (for example, collections) need access to the same content. Multiple pods can have access to the same persistent volume claim (PVC) as long as you select the `ReadWriteMany` storage option.
+
+In addition, OpenShift Data Foundation provides a `ReadWriteMany` or S3-compliant implementation. Also, an NFS storage configuration can be setup to support `ReadWriteMany` and is often the simplest approach. This, however, introduces the NFS server as a potential single point of failure.
```

Comment:
Why do we mention Data Foundation here? Should we use the variable instead `{ODFShort}`?

Reply:
"simplest approach" compared to what? I'm not sure what this is telling the reader to do with storage. Can you use ODF as the underlying storage mechanism?

There's also passive voice in the second sentence "can be setup". That should be a phrasal verb rather than noun form as well. s/setup/set up

Reply:
I just added OpenShift Data Foundation to the attributes file yesterday, per @ariordan-redhat 's request. So this is now updated.